src/http/server.ts

```diff
@@ -257,7 +269,19 @@ export class Server {
       if (typeof resource[request.method.toUpperCase()] !== "function") {
         throw new Drash.Exceptions.HttpException(405);
       }
-      response = await resource[request.method.toUpperCase()]();
+      response = await resource[request.method.toUpperCase()](); // response can  be literally anything, it's down to the user what they return from the method
```

Comment:
can you put the comment above this line? (nitpicky i know... sorry!)

Reply:
https://github.com/drashland/deno-drash/pull/412/commits/18a790908aae005f3c483cecf3bdb04349a42958